examples/configuring-via-cli/configuring_via_cli.md

```diff
@@ -0,0 +1,12 @@
+# Configuring via command line arguments
+
+The values of the configurable variables can be configured through the command line arguments when executing the Ballerina program. The provided value is expected to be the `toString()` representation of the intended value.
+
+The command-line-based configuration is only supported for configurable variables of types `int`, `byte`, `float`, `boolean`, `string`, `decimal`, `enum`, and `xml`.
+The syntax `-Ckey=value` can be used to provide values through the command line parameters.
+
+For more information, see [Configure via command line arguments](/learn/configure-ballerina-programs/provide-values-to-configurable-variables/#provide-via-command-line-arguments/).
```

Comment:
```suggestion
For more information, see [Configure via command-line arguments](/learn/configure-ballerina-programs/provide-values-to-configurable-variables/#provide-via-command-line-arguments/).
```
We need to update the word `command-line` in the learn page also.

Reply:
Created an issue for this https://github.com/ballerina-platform/ballerina-dev-website/issues/6194. I'll address in a separate PR.